packages/api/src/controllers/stripe.ts

```diff
@@ -379,6 +453,30 @@ app.post("/migrate-personal-user", async (req, res) => {
           `User ${user.email} has ${subscriptionItems.data.length} subscription items`
         );
 
+        let payAsYouGoItems = [];
+
+        const payAsYouGoPrices = await req.stripe.prices.list({
+          lookup_keys: products["pay_as_you_go_1"].lookupKeys,
+        });
+        payAsYouGoItems = payAsYouGoPrices.data.map((item) => ({
+          price: item.id,
+        }));
+
+        if (!req.body.actually_migrate) {
+          res.status(500).json({
```

Comment:
Did you really mean a 500 here? Can be confusing when testing dry run
```suggestion
          res.json({
```